corehq/apps/zapier/views.py

```diff
@@ -155,4 +156,4 @@ def post(self, request, *args, **kwargs):
             update=properties
         )
 
-        return HttpResponse('OK')
+        return HttpResponse('Case ' + case_id + " updated")
```

Comment:
nit: the [more pythonic](http://stackoverflow.com/a/10043957/2957657) way of concatenating strings together is to use the `format` function:
```py
"Case {case_id} updated".format(case_id=case_id)
```

(applies to all strings in this PR)